packages/jaeger-ui/src/model/link-patterns.tsx

```diff
@@ -157,12 +158,34 @@ export function computeTraceLink(linkPatterns: ProcessedLinkPattern[], trace: Tr
   return result;
 }
 
+function getParameterValues(
+  parameters: string[],
+  items: KeyValuePair[],
+  findInAncestorsFn?: (_: string) => KeyValuePair | undefined,
+  onMissingFn?: (_: string) => void
+) {
+  const parameterValues: Record<string, any> = {};
+  const allParameters = parameters.every(parameter => {
```

Comment:
```suggestion
  parameters.forEach(parameter => {
```

To be able to render different paraments(from different scopes. HTTP calls statuses and Redis's driverID, for example)

Reply:
This code is from @everett980 at https://github.com/jaegertracing/jaeger-ui/pull/487/files#r356227730.

What happens to return values if I make that change?
Are you proposing that the "all elements match otherwise nothing" behaviour be changed?
That was quite deliberate in the original PR; @lookfwd [added a test for it](https://github.com/jaegertracing/jaeger-ui/pull/487/files#diff-5366cdb1f6c0803f57f6dc374b480ba6abc8897a82ab0ab7bc517532bd2d5ec9R95)